src/Microsoft.Health.Fhir.Liquid.Converter.Tool/TemplateManagementLogicHandler.cs

```diff
@@ -6,115 +6,80 @@
 using System;
 using System.IO;
 using System.Threading.Tasks;
-using Microsoft.Extensions.Logging;
 using Microsoft.Health.Fhir.Liquid.Converter.Tool.Models;
 using Microsoft.Health.Fhir.TemplateManagement;
-using Microsoft.Health.Fhir.TemplateManagement.Exceptions;
-using Newtonsoft.Json;
 
 namespace Microsoft.Health.Fhir.Liquid.Converter.Tool
 {
     internal static class TemplateManagementLogicHandler
     {
-        private static readonly ILogger Logger;
-        private static readonly TextWriter ErrorWriter;
-
-        static TemplateManagementLogicHandler()
-        {
-            using var loggerFactory = LoggerFactory.Create(builder =>
-            {
-                builder.AddFilter("Microsoft.Health.Fhir.TemplateManagement", LogLevel.Trace)
-                       .AddConsole();
-            });
-            FhirConverterLogging.LoggerFactory = loggerFactory;
-            Logger = FhirConverterLogging.CreateLogger(typeof(TemplateManagementLogicHandler));
-
-            ErrorWriter = Console.Error;
-        }
-
-        internal static async Task PullAsync(PullTemplateOptions options)
+        internal static async Task<int> PullAsync(PullTemplateOptions options)
         {
             try
             {
                 if (!options.ForceOverride)
                 {
                     if (Directory.Exists(options.OutputTemplateFolder) && Directory.GetFileSystemEntries(options.OutputTemplateFolder).Length != 0)
                     {
-                        ExceptionHandling(options.ErrorJsonFile, new Exception("The output folder is not empty. If force to override, please add -f in parameters"), "Process Exits: ");
-                        return;
+                        Console.Error.WriteLine($"Fail to pull templates: The output folder is not empty. If force to override, please add -f in parameters");
+                        return -1;
                     }
                 }
 
                 OCIFileManager fileManager = new OCIFileManager(options.ImageReference, options.OutputTemplateFolder);
                 if (await fileManager.PullOCIImageAsync())
                 {
                     fileManager.UnpackOCIImage();
-                    Logger.LogInformation($"Succeed to pull templates to {options.OutputTemplateFolder} folder");
+                    Console.WriteLine($"Succeed to pull templates to {options.OutputTemplateFolder} folder");
+                    return 0;
                 }
                 else
                 {
-                    ExceptionHandling(options.ErrorJsonFile, new TemplateManagementException("Fail to pull templates."), "Process Exits: ");
+                    Console.Error.WriteLine($"Fail to pull templates.");
```

Comment:
Do we expect there is an "Process Exits: " prefix when there are errors? If so we could be consistent here.

Reply:
Only use this prefix when throw input/output folders exception before pull/push operations.